modules/launcher/src/main/java/org/mule/module/launcher/DomainDeploymentListener.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.launcher;
+
+/**
+ * Defines a listener for deployment events for Mule domains.
+ */
+public interface DomainDeploymentListener extends DeploymentListener
```

Comment:
Why do you need this interface? Listening for domain notifications can be done using the new API and the adapter only knows about DeploymentListener